Snakefile

```diff
@@ -72,6 +73,26 @@ if "builds" not in config:
 
 include: "workflow/snakemake_rules/reference_build_definitions.smk"
 
+# Check for old-style input file references and alert users to the new format.
+if "sequences" in config or "metadata" in config:
+    logger.error("ERROR: Your configuration file includes references to a deprecated specification of input files (e.g., `config['sequences']` or `config['metadata']`).")
+    logger.error("Update your configuration file (e.g., 'builds.yaml') to define your inputs as follows and try running the workflow again:")
+    logger.error(textwrap.indent(
+        f"\ninputs:\n  metadata: {config['metadata']}\n  sequences: {config['sequences']}\n",
```

Comment:
This error message is really helpful! Two points:

1. Suggesting the new syntax is a wonderful addition! However it's missing the `name` key, and won't work as currently written.
2. (pedantic) I think this input syntax is "no longer supported" rather than "deprecated", as deprecated functionality should still be possible to use. (At least, that's my understanding of the word.)

Reply:
Fixed!